GeoCAT-comp-examples/README.rst

```diff
@@ -0,0 +1,22 @@
+.. _comp-examples-index:
+
+.. _gallery-geocat-comp:
+
+====================
+GeoCAT-comp Examples
+====================
+
+This gallery contains usage examples for the GeoCAT-comp functions.
+These examples are represented as Python scripts and their
+corresponding Jupyter notebooks, which may or may not contain
+visualizations (i.e. Matplotlib plots), depending on the function
+characteristics.
+
+For visualization, mainly `matplotlib` and `cartopy` are used. In addition,
+`geocat-datafiles <https://github.com/NCAR/geocat-datafiles>`_ is used as a
+dataset storage, `geocat-viz <https://github.com/NCAR/geocat-viz>`_ is used for
+a higher level implementation for low level `matplotlib` functionalities, as
+well as `xarray` and `numpy` are used for data processing.
```

Comment:
```suggestion
For visualization, mainly `matplotlib` and `cartopy` are used. In addition,
`geocat-datafiles <https://github.com/NCAR/geocat-datafiles>`_ is used as a
dataset storage and `geocat-viz <https://github.com/NCAR/geocat-viz>`_ is used for
a higher level implementation for low level `matplotlib` functionalitie., `Xarray` and `numpy` are used for data processing.
```

Just a little bit of rewording. Is this paragraph necessary though? I assume if more geocat-comp examples are added, this list will need to be expanded and I'm just not sure we need to enumerate some of the packages that we used like this.

Reply:
Thanks for the rewording and question! I intended this paragraph to give general information about our visualization and data processing implementations, and what significant packages we are using. That said, whatever the number of new GeoCAT-comp examples, I would expect this paragraph to stay same. Thoughts?